docs/mermaid_diagrams.rst

```diff
@@ -0,0 +1,207 @@
+.. _mermaid.js: https://mermaid-js.github.io/mermaid/
+
+Mermaid diagrams
+================
+
+.. note::
+    Use of this feature has no affect or affiliation with sphinx's graphviz implementation.
+
+The mkdocs-material theme is equipped to make use of diagrams generated (during page load time)
```

Comment:
Perhaps put this as a `.. note` at the bottom, since mkdocs-material's implementation isn't terribly relevant to users of this theme, except I suppose if they are migrating.

Reply:
Its more of an introductory note. I could hide it into a collapsible admonition titled "Implementation Note".

There is a similar introductory statement in practically all supported MD extensions (in mkdocs) that get special attention for this port of the theme. This was added specifically for those migrating from mkdocs.

Reply:
I'll leave this up to your judgement then.

Reply:
I favor leaving it as is because a collapsible admonition will still draw attention (even if collapsed by default). If the implementation detail was much longer or more specific, then it would be a good idea to hide that in a collapsible admonition.

Notice that this paragraph exists before the directive's docs, so a link directly to the directive's docs will scroll past the introductory note(s).

Reply:
That and I don't want to retrofit all the other statements that talk about the differences between upstream and this port.